CuratedContent/InclusivityBugs.md

```diff
@@ -0,0 +1,93 @@
+Author(s) | Zoe Steine-Hanson, Claudia Hilderbrand, Lara Letaw, Jillian Emard, Christopher Perdriau, Christopher Mendez, Margaret Burnett, Anita Sarma
+Focus | Best Practices
+
+Discusses inclusivity bugs arising out of gender bias.
+
+Resource information | Details
+:--- | :---
+Article title | [Squashing inclusivity bugs in open source software](https://opensource.com/article/18/8/inclusivity-bugs-open-source-software)
+Author(s) | Anita Sarma
+Focus | Best Practices
+
+Methodology for identifying gender bias in software tools to help eliminate it.
+
+Resource information | Details
+:--- | :---
+Article title | [How to Debug Inclusivity Bugs? An Empirical Investigation of Finding-to-Fixing with Information Architecture](https://ir.library.oregonstate.edu/concern/defaults/8049gc556)
+Author(s) | Guizani, Mariam; Steinmacher, Igor; Emard, Jillian; Fallatah, Abrar; Burnett, Margaret; Sarma, Anita
+Focus | Experience Report
+
+Discusses some of the bug *fixes* applied to address inclusivity bugs related to gender bias.
+
+Resource information | Details
+:--- | :---
+Article title | [AID: An Automated Detector for Gender-Inclusivity Bugs in OSS Project Pages](https://ieeexplore.ieee.org/document/9402060)<br>[DOI:10.1109/ICSE43902.2021.00128](10.1109/ICSE43902.2021.00128)
+Author(s) | Amreeta Chatterjee; Mariam Guizani; Catherine Stevens; Jillian Emard; Mary Evelyn May; Margaret Burnett; Iftekhar Ahmed
+Focus | Tooling
+
+Discusses a automated tool for finding inclusivity bugs related to gender bias.
+
+Resource information | Details
+:--- | :---
+Article title | [Engineering Gender-Inclusivity into Software: Ten Teams’ Tales from the Trenches](https://par.nsf.gov/servlets/purl/10226803)
+Author(s) | Claudia Hilderbrand, Christopher Perdriau, Lara Letaw, Jillian Emard, Zoe Steine-Hanson, Margaret Burnett, Anita Sarma
+Focus | Inclusive Design
+
+Various teams experiences designing for inclusivity.
+
+Resource information | Details
+:--- | :---
+Podcast title | [How Men and Women Approach Open Source Differently](https://thenewstack.io/how-men-and-women-approach-open-source-differently/)
+Speaker(s) | Anita Sarma
+Focus | Inclusive Communication
+
+Discusses how gender differences lead to differences in approach to open source software contributions.
+
+Resource information | Details
+:--- | :---
+Article title | [How to Apply Metrics for Inclusion to your Open Source Project](https://medium.com/@sunnydeveloper/how-to-apply-metrics-for-inclusion-to-your-open-source-project-71b4e31a7b0c)
+Author(s) | Emma Irwin
+Focus | Productivity Metrics
+
+A quick look at some inclusivity metrics for open source projects.
+
+Resource information | Details
+:--- | :---
+Article title | [Inclusivity Bugs and The Language We Use](https://www.computer.org/csdl/magazine/cs/2021/06/09658239/1zw1kDaA63e)<br> [DOI:10.1109/MCSE.2021.3125473](10.1109/MCSE.2021.3125473)
```

Comment:
```suggestion
Article title | [Inclusivity Bugs and The Language We Use](https://www.computer.org/csdl/magazine/cs/2021/06/09658239/1zw1kDaA63e)<br> [DOI:10.1109/MCSE.2021.3125473](https://ieeexplore.ieee.org/document/9658239)

```


Maybe it's just me, the link doesn't work when I click it. I added a different link that might work for more people ?